app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info.get("expiration_date")
+
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine is outdated")
+
+        if "wearing_a_mask" not in visitor or not visitor["wearing_a_mask"]:
```

Comment:
Code Style: Avoid using `not in` operator twice. You can simplify this condition by using the `get` method of the dictionary which will return `None` if the key is not present in the dictionary. Here is how you can do it:
```python
if not visitor.get('wearing_a_mask'):
    raise NotWearingMaskError('Visitor is not wearing a mask')
```